api/routes/user.js

```diff
@@ -244,23 +244,51 @@ router.post('/search', function (req, res) {
           joinDate: result.joinDate,
           lastLogin: result.lastLogin,
           membershipValidUntil: result.membershipValidUntil,
-          pagesPrinted: result.pagesPrinted
+          pagesPrinted: result.pagesPrinted,
+          doorCode: result.doorCode
         }
         return res.status(OK).send(user)
       })
     }
   })
 })
 
+// Search for all members
+router.post('/users', function (req, res) {
```

Comment:
```suggestion
router.get('/users', function (req, res) {
```

Reply:
negative, post is more secure and i dont want to deal with http params

Reply:
also router.post is our api format and I don't want to deviate from that too much